### PR TITLE
Add functionality to get git release info using acikubectl

### PR DESCRIPTION
**Pull request by apoorva11029**
To get the git commit ID and build time, run on any host:
acikubectl version
or
acikubectl version --kubeconfig=required_path

Git commit info is saved inside the aci-containers-controller container  and can be accessed by running "aci-containers-controller --version"

### CODE REVIEW

File: cmd/acikubectl/cmd/version.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// Copyright © 2017 Cisco Systems, Inc.
```
Comment (jojimt) [inline cmd/acikubectl/cmd/version.go]: 2019 :)

File: cmd/acikubectl/cmd/version.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        for _, pod := range systemNamespacePods.Items {
+                if strings.Contains(pod.Name, "aci-containers-controller") {
+                    buffer := new(bytes.Buffer)
+                    mylist := []string{"exec", "-n" + systemNamespace, pod.Name, "--", "/bin/sh", "-c", "get-gitCommit"}
+                    execKubectl(mylist, buffer)
+                    fmt.Println(buffer.String())
+                    break
+                }
+        }
+        return
```
Comment (jojimt) [inline cmd/acikubectl/cmd/version.go]: Please eliminate the redundant return.

File: pkg/controller/controller.go [hunk 3]
```diff
@@ -274,6 +275,19 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	var err error
 	var privKey []byte
 	var apicCert []byte
+
```
Comment (jojimt) [inline pkg/controller/controller.go]: Please move this to cmd/controller/main.go. If you add --version as a cli flag there, you can also eliminate the shell script. Instead, you could just invoke controller --version which would just print the version and exit.

File: scripts/getGitCommit.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if command -v git &>/dev/null && git rev-parse &>/dev/null; then
+    GIT_COMMIT=$(git rev-parse HEAD)
```
Comment (jojimt) [inline scripts/getGitCommit.sh]: Can you add these lines as well? 
        if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
                GIT_COMMIT="$GIT_COMMIT-unsupported"
        fi

File: cmd/acikubectl/cmd/version.go (hunk 2 above)
Comment (jojimt) [reply cmd/acikubectl/cmd/version.go]: Also, please run gofmt -s -w on all modified go files.

File: cmd/acikubectl/cmd/version.go (hunk 1 above)
Comment (apoorva11029) [reply cmd/acikubectl/cmd/version.go]: Done, thanks for reminding!

File: cmd/acikubectl/cmd/version.go (hunk 2 above)
Comment (apoorva11029) [reply cmd/acikubectl/cmd/version.go]: Added in the latest commit

File: pkg/controller/controller.go (hunk 3 above)
Comment (apoorva11029) [reply pkg/controller/controller.go]: Thanks for the suggestion, made life easier. Removed the script and added the --version cli flag to aci-containers-controller

File: scripts/getGitCommit.sh (hunk 4 above)
Comment (apoorva11029) [reply scripts/getGitCommit.sh]: Not adding it here because our current build leaves some untracked files

File: cmd/acikubectl/cmd/version.go [hunk 5]
```diff
@@ -0,0 +1,66 @@
+// Copyright © 209 Cisco Systems, Inc.
+//
```
Comment (jojimt) [inline cmd/acikubectl/cmd/version.go]: Typo -- should be 2019

Comment (apoorva11029) [reply cmd/acikubectl/cmd/version.go]: Yes thanks, fixed.

**Pull request merged**